x-pack/plugins/observability_solution/observability/public/components/custom_threshold/helpers/threshold_unit.test.ts

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  Aggregators,
+  CustomThresholdExpressionMetric,
+} from '../../../../common/custom_threshold_rule/types';
+import { convertToApiThreshold } from './threshold_unit';
+
+describe('convertToApiThreshold', () => {
```

Comment:
These are the same tests that we had before in [adjust_threshold_based_on_format.test.ts](https://github.com/elastic/kibana/pull/179841/files#diff-d193773325cf5d72dc1b864baebc378a09b3eb7f62c76d1962c860dee65c8127), just added 3 more tests at the end and renamed the file.